src/ResourceManager/Network/Commands.Network/Models/PSApplicationGatewayAvailableWafRuleSetsResult.cs

```diff
@@ -0,0 +1,31 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public class PSApplicationGatewayAvailableWafRuleSetsResult
+    {
+        public List<PSApplicationGatewayFirewallRuleSet> Value { get; set; }
+
+        [JsonIgnore]
+        public string ValueText
```

Comment:
@jobatzil for each of the classes you are adding to the `Model` folder, can you make sure that each of their properties have corresponding help with them?

```cs
/// <summary>
/// Brief summary of property.
/// </summary>
public string ValueText
{
    ...
}